pkg/master/controller.go

```diff
@@ -140,14 +141,33 @@ func (c *Controller) CreateNamespaceIfNeeded(ns string) error {
 	return err
 }
 
+// createPortAndServiceSpec creates an array of service ports.
+// If the NodePort value is 0, then this signifies the standard TCP port.
```

Comment:
Please reword, this uses the TCP protocol in both cases, and servicePort in the normal case.